app/infrastructure/services/webhook_dispatcher.py

```
@@ -0,0 +1,162 @@
+        ],
+        *,
+        http_client: httpx.AsyncClient | None = None,
+    ) -> None:
+        """get_subscriptions(tenant_id) -> list[WebhookSubscriptionForDispatch] (async)."""
+        self._get_subscriptions = get_subscriptions
+        self._http = http_client
+
+    async def dispatch(
+        self,
+        tenant_id: str,
+        event: "EventEntity",
```

Ruff (0.15.5)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)